core/db_models/EEM_Payment_Method.model.php

```diff
@@ -27,82 +24,97 @@ class EEM_Payment_Method extends EEM_Base
     const scope_api = 'API';
 
     /**
-     *
      * @type EEM_Payment_Method
      */
     protected static $_instance = null;
 
 
-
     /**
      * private constructor to prevent direct creation
+     *
      * @Constructor
      * @access   protected
-     * @return EEM_Payment_Method
+     * @param null $timezone
+     * @throws EE_Error
      */
     protected function __construct($timezone = null)
     {
-        $this->singlular_item = __('Payment Method', 'event_espresso');
+        $this->singular_item = __('Payment Method', 'event_espresso');
```

Comment:
singlular ???

Reply:
singlular from single

Reply:
point taken... English often makes little to no sense.

Reply:
Same thing on https://github.com/eventespresso/event-espresso-core/blob/97c97dfe9e55dd6115abb6026f9ba30164a5cbea/core/db_models/EEM_Event_Message_Template.model.php#L28 ain't it ?

Reply:
good catch... will fix